boards/arm/v2m_musca/v2m_musca-nonsecure.dtsi

```diff
@@ -0,0 +1,59 @@
+/ {
+	flash0: flash@230400 {
+		/* QSPI flash */
+		reg = <0x00230400 0x1ffc00>;
+	};
+
+	sram0: memory@20010000 {
+		compatible = "mmio-sram";
+		reg = <0x20010000 0x10000>;
```

Comment:
Same comment here, really.  The whole sram needs to be described.  We need an application offset for this to point to the correct memory partition.  So this reg should be <0x20000000 0x20000>.  And we need something to express the application offset.  Maybe reserved-memory?

Reply:
I agree with @agross-linaro: .dtsi should describe the whole flash and sram of the SOC. However, **somewhere** (in DTS, ideally)  we need to define which will be the SRAM to be used by the NONSECURE firmware image. 

So we need something similar to the flash partitioning, where we "choose" slot-1 instead of all-flash, when we have MCUboot present.